letsencrypt/client/auth_handler.py

```diff
@@ -201,9 +203,7 @@ def _assign_responses(self, flat_list, ichall_dict):
 
     def _path_satisfied(self, dom):
         """Returns whether a path has been completely satisfied."""
-        return all(
-            None != self.responses[dom][i] and "null" != self.responses[dom][i]
-            for i in self.paths[dom])
+        return all(self.responses[dom][i] is not None for i in self.paths[dom])
```

Comment:
same as below... I didn't take into account the False parameters... but "null" is an incomplete challenge along the path.